org.lflang/src/org/lflang/generator/ts/TSReactionGenerator.kt

```diff
@@ -119,27 +124,28 @@ class TSReactionGenerator(
                 }
             }
         }
+
         return with(PrependOperator) {
             """
             |
-            |this.addReaction(
+            |this.add${if (reaction.isMutation()) "Mutation" else "Reaction"}(
             |    new __Triggers($reactionTriggers),
-            |    new __Args($reactFunctArgs),
+            |    new __Args($reactFuncArgs),
             |    function ($reactSignature) {
             |        // =============== START react prologue
-        ${" |        "..reactPrologue}
+            ${" |        "..reactPrologue}
```

Comment:
The indentation here is weird. Same on line 139, 142, 145, etc.

Reply:
https://github.com/lf-lang/lingua-franca/pull/1318/commits/e07dc3972dac5adfd4a7e864607f6f746cb6f049